service/dash.go

```diff
@@ -153,6 +161,29 @@ func generateMPD(outputPath string) {
 	log.Println(string(o))
 }
 
+// Returns input based on if the watermark is needed or not
+func getInput(targetFile string, watermarkFile string, withWatermark bool, watermarkDimension map[string]int, watermarkPosition map[string]int) *ffmpeg.Stream {
+
+	input := ffmpeg.Input(targetFile)
+
+	if withWatermark {
+		filterArgs := "" + strconv.Itoa(watermarkPosition[constants.WaterMarkPositionMap[constants.X]]) + ":" + strconv.Itoa(watermarkPosition[constants.WaterMarkPositionMap[constants.Y]]) + ""
```

Comment:
can you use the argument generator in utils package

Reply:
The output is not what is expected as input

The output of argument generator is ```-64 1 ```
The output required is ```[-64:1]```

Reply:
got it